jx_sqlite/aggs_table.py

```diff
@@ -360,12 +360,21 @@ def _get(row):
                     column_number = len(outer_selects)
 
                     for json_type, sql in details.sql.items():
-                        concat_sql.append("GROUP_CONCAT(QUOTE(DISTINCT(" + sql + ")))")
+                        concat_sql.append("GROUP_CONCAT(DISTINCT(" + sql + "))")
```

Comment:
the `test_union_multiple` is testing that two separate columns are isolated.  First, check that the result coming out of the database is correct.